### PR TITLE
Merge/merging release 2.25 into develop ver 3

### PR DESCRIPTION
**Pull request by dk1844**
- locally & on-Jenkins buildable
### Testrun
 - has been test-run with `-Dspline.mode=DISABLED`, but since there haven't been spline updates, it should be a problem.

### CODE REVIEW

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala [hunk 1]
```diff
@@ -22,6 +22,8 @@ import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.junit4.SpringRunner
 import za.co.absa.enceladus.model.conformanceRule.MappingConformanceRule
+import za.co.absa.enceladus.model.conformanceRule.MappingConformanceRule
```
Comment (AdrianOlosutean) [inline rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala:25]: Dulplicates

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/DatasetRepositoryIntegrationSuite.scala [hunk 2]
```diff
@@ -25,6 +25,7 @@ import za.co.absa.enceladus.model.Dataset
 import za.co.absa.enceladus.rest_api.exceptions.EntityAlreadyExistsException
 import za.co.absa.enceladus.rest_api.integration.fixtures.{DatasetFixtureService, FixtureService}
 import za.co.absa.enceladus.rest_api.repositories.DatasetMongoRepository
+import za.co.absa.enceladus.model.Dataset
```
Comment (AdrianOlosutean) [inline rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/DatasetRepositoryIntegrationSuite.scala:28]: Duplicate

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/StatisticsIntegrationSuite.scala [hunk 3]
```diff
@@ -21,6 +21,7 @@ import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.junit4.SpringRunner
 import za.co.absa.enceladus.rest_api.integration.fixtures.{DatasetFixtureService, FixtureService, PropertyDefinitionFixtureService}
+import za.co.absa.enceladus.rest_api.repositories.{DatasetMongoRepository, PropertyDefinitionMongoRepository}
```
Comment (AdrianOlosutean) [inline rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/StatisticsIntegrationSuite.scala:24]: Not needed

**Review by AdrianOlosutean — COMMENTED** [review]
Just some small things

File: menas/ui/service/PropertiesService.js [hunk 4]
```diff
@@ -21,6 +21,10 @@ class PropertiesDAO {
   static getProperty(propertyName) {
     return RestClient.get(`/properties/datasets/${propertyName}`);
   }
+
+  static getProperty(propertyName) {
+    return RestClient.get(`api/properties/datasets/${propertyName}`);
```
Comment (AdrianOlosutean) [inline menas/ui/service/PropertiesService.js:26]: This is breaking
```suggestion
    return RestClient.get(`properties/datasets/${propertyName}`);
```

**Review by AdrianOlosutean — CHANGES_REQUESTED** [review]

File: menas/ui/service/PropertiesService.js (hunk 4 above)
Comment (dk1844) [reply menas/ui/service/PropertiesService.js:26]: Thanks for noticing!

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala (hunk 1 above)
Comment (dk1844) [reply rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala]: Removed, thanks for noticing.

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/DatasetRepositoryIntegrationSuite.scala (hunk 2 above)
Comment (dk1844) [reply rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/DatasetRepositoryIntegrationSuite.scala:27]: import za.co.absa.enceladus.rest_api.repositories.{DatasetMongoRepository, PropertyDefinitionMongoRepository}

File: rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/StatisticsIntegrationSuite.scala (hunk 3 above)
Comment (dk1844) [reply rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/StatisticsIntegrationSuite.scala]: Removed, thanks for noticing.

**Review by AdrianOlosutean — APPROVED** [review]
Tested locally, looks good

**Pull request merged**